sdk/com.ibm.sbt.web/src/main/webapp/js/sdk/sbt/xpath.js

```diff
@@ -28,7 +28,7 @@ define(['./declare'],function(declare) {
 		constructor: function(xpath, nsArray){
 		     this.xpath = xpath;    
 		     this.nsArray = nsArray || {};
-		     if (!document.evaluate) {
+		     if (!document.evaluate && !navigator.getUserMedia) { 
```

Comment:
why aren't you using the user-agent string 
Pre Microsoft Edge you can detect MSIE ??.?
Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.1; WOW64; Trident/7.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; .NET4.0C; .NET4.0E)

For Microsoft Edge, it shouldn't matter if it uses the processing from mozilla. 

eg !(navigator.userAgent.lastIndexOf("MSIE") === 0)